frontend/src/containers/EditDashboard.tsx

```diff
@@ -234,17 +234,6 @@ function EditDashboard() {
                 {t("PublishButton")}
               </Button>
             </span>
-            <Tooltip
```

Comment:
I think we should move this a usa-sr-only text and not remove it

Reply:
the suggestion has been implemented.

Reply:
looks great!